packages/rocketchat-ui-message/client/messageBox.coffee

```diff
@@ -165,7 +166,7 @@ Template.messageBox.events
 				e.preventDefault()
 				files.push
 					file: item.getAsFile()
-					name: 'Clipboard'
+					name: 'Clipboard - ' + moment().format('L, LTS')
```

Comment:
Hey @karlprieb, thanks for the feedback.
How do you feel about using the `Message_TimeAndDateFormat` for this? I think it would be better for searching through older files. I'll add a screenshot to the PR.